docs/data-sources/service_accounts.md

```diff
@@ -27,10 +27,10 @@ data "twingate_service_accounts" "foo" {
 
 ### Read-Only
 
-- `id` (String) The ID of this resource.
-- `service_accounts` (Block List) List of Service Accounts (see [below for nested schema](#nestedblock--service_accounts))
+- `id` (String) The ID of the Service Accounts datasource.
```

Comment:
Let's change this string to this: "The ID of the data source result"

You can use this on all data sources.